ros_buildfarm/status_page.py

```diff
@@ -1030,3 +1034,39 @@ def _compare_package_version(distros, pkg_name):
         data[i] = '<div>%s</div>' % value
 
     return data
+
+
+REPOS_DATA_NAMES = ['build', 'test', 'main']
+def write_summary(summary_filename, ordered_pkgs, repos_data):
+    print("Generating status summary '%s':" % summary_filename)
+    summary = {}
+    for pkg in ordered_pkgs:
+        pkg_d = {'version': pkg.version, 'url': pkg.repository_url, 'status': pkg.status}
+        if pkg.status_description:
+            pkg_d['status_description'] = pkg.status_description
+        pkg_d['maintainers'] = []
+        for maintainer in pkg.maintainers:
+            maintainer_dict = {'email': maintainer.email}
+            try:
+                maintainer_dict['name'] = str(maintainer.name)
+            except:
+                maintainer_dict['name'] = maintainer.name
```

Comment:
Can you describe why the `try` / `except` is needed (potentially adding a comment to the code to clarify)?